gradle.properties

```diff
@@ -5,4 +5,4 @@ group = com.enonic.app
 projectName = metafields
 appName = com.enonic.app.metafields
 xpVersion = 7.0.0
-version=1.10.0-SNAPSHOT
+version=1.10.1--SNAPSHOT
```

Comment:
too many dashes?

Reply:
and this should be "1.11.0-SNAPSHOT", since this is a feature release, not a bug fix.

Reply:
Yeah this is allready a feature: https://github.com/enonic/app-metafields/commit/1da6690a0643b7758ba6f85388a85af5816d70c1 
The version fix is the next version 1.10.1. The one that will be on the market will be v1.10

Will fix to many dashes

Reply:
This pull request includes a version update: https://github.com/enonic/app-metafields/pull/59/commits/ea72299988a166907ed9bdb348821d099b7c2070 
That commit is what i will use to build and publish on the market